index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
+            found, or if subclass does not extend AudioWorkletProcessor, throw
+            an exception from the node constructor.
+            <ol>
+              <li>
+                Start registered constructor for custom processor subclass,
+                passing structured clone of the options object that was provided
+                to AudioWorkletNode.
+              </li>
+              <li>
+                Start super constructor for AudioWorkletProcessor.
+              </li>
+              <li>
+                Initialize the processor’s states.
+              </li>
+              <li>
+                End the construction of an AudioWorkletProcessor and return to
+                AudioWorkletNode constructor.
+              </li>
+            </ol>
+          </li>
+          <li>
+            <em>In the window scope</em>: Construct AudioParam objects in the node
+            based on the information from the processor’s
+            parameterDescriptors getter and initialize parameter values from
+            options where the keys match names declared in
+            AudioParamDescriptors.
+          </li>
+          <li>
+            <em>In the window scope</em>: Return from the AudioWorkletNode
+            constructor.
+          </li>
+        </ol>
+        <section>
+          <h2 id="AudioWorkletNode">
+            The <dfn>AudioWorkletNode</dfn> interface
+          </h2>
+          <p>
+            This interface represens an <a>AudioNode</a> which lives on the main
+            control thread. The user can create this node from an instance of
+            <a>AudioContext</a> and such node can be connected with other built-in
+            <a>AudioNode</a>s to form an audio graph.
+          </p>
+          <dl title=
+          "[Constructor(AudioContext context, DOMString name, optional AudioWorkletNodeOptions options)] interface AudioWorkletNode : AudioNode" 
+          class="idl">
+            <dt>
+              attribute EventHandler onmessage
+            </dt>
+            <dd>
+              <p>
+                The onmessage handler is called whenever the associated
+                <a>AudioWorkletProcessor</a> posts a message back to the main
+                thread.
+              </p>
+            </dd>
+            <dt>
+              void postMessage(any message, optional sequence&lt;Transferable&gt;
+              transfer)
+            </dt>
+            <dd>
+              <p>
+                postMessage May be called to send a message to the corresponding
+                <a>AudioWorkletProcessor</a> via the algorithm defined by the
+                Worker specification.
+              </p>
+            </dd>
+          </dl>
+
+          <section>
+            <h2 id="AudioWorkletNodeOptions">
+              AudioWorkletNodeOptions
+            </h2>
+            <p>
+              The AudioNodeOptions dictionary can be used for the custom
```

Comment:
AudioNodeOptions -> `<a><code>AudioNodeOptions</code></a>`

Did you really mean AudioNodeOptions and not AudioWorkletNodeOptions?